packages/stripe-integration/src/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -323,11 +331,15 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
 
             return await this.paymentIntegrationService.submitPayment(paymentPayload);
         } catch (error) {
-            return this._processVaultedAdditionalAction(
-                error,
-                methodId,
-                shouldSetAsDefaultInstrument,
-            );
+            if (error instanceof Error) {
+                return this._processVaultedAdditionalAction(
```

Comment:
Any error should be passed to `_processVaultedAdditionalAction` method, so might be better to update error interface in options list of the method. 

cc @PavlenkoM for visibility

Reply:
Changed signature in order to remove this type check.
```
_processVaultedAdditionalAction(
        error: Error,
        error: unknown,
        methodId?: string,
        shouldSetAsDefaultInstrument = false,
    )
```